src/wp-includes/update.php

```diff
@@ -980,3 +1036,5 @@ function wp_clean_update_cache() {
 add_action( 'wp_maybe_auto_update', 'wp_maybe_auto_update' );
 
 add_action( 'init', 'wp_schedule_update_checks' );
+
+add_action( 'wp_delete_temp_updater_backups', 'wp_delete_all_temp_backups' );
```

Comment:
Does this have the correct callback name? On mobile but looks like it needs the leading  `_`?

Reply:
I’m an idiot

Reply:
That's for the cron task.